huh/db.py

```diff
@@ -81,6 +81,53 @@ def all(cls, conn: sqlite3.Connection):
         res = cur.execute(f"SELECT rowid AS id, * FROM {cls.table_name}", ())
 
         return (cls(*row) for row in res)
+    
+    @classmethod
+    def all_ann(cls, conn):
```

Comment:
this method is not necessary once you move all the methods to the announcement class, use ``Announcement.all()`` instead

Reply:
oh I kinda need the name of the user also tho

Reply:
you can get the id of the user on the api layer via ``current_user.id``.

eg.
```python
from flask_login import login_required, current_user
from flask import render_template, Blueprint, request
from huh.db import Comment, connect

bp = Blueprint("comment", __name__, url_prefix="/comment")

@bp.route("/create", methods=["POST"])
@login_required
def create():
    author_id = current_user.id
    # rest of ur code

```